.github/workflows/update_build_status.yml

```diff
@@ -0,0 +1,54 @@
+name: Update build status
+
+on:
+  schedule:
+  - cron: "*/15 * * * *"
```

Comment:
Just a question. What happens when the fork disables this job?
GitHub allows us to disable workflows. So, I usually disables the snapshot publishing jobs.

Reply:
Yeah, that's a problem. I will try to integrate with ["action required" status](https://docs.github.com/en/rest/guides/getting-started-with-the-checks-api#about-check-suites) of the checks in GitHub Actions (that allows us to link URLs with some guides)

This workflow is okay because this one is supposed to run in the main repo but if users disable the main build job, it's a problem.

This is one example when a contributor did not enable the main build job: https://github.com/apache/spark/pull/32187